subjects/mobile-dev/chess/resources/README.md

```diff
@@ -0,0 +1,109 @@
+# ♟️ chess websocket server
+
+This project represents a simple websocket server to play chess.
+
+- [Local setup](#local-setup)
+- [Interacting with websocket server to play chess](#interacting-with-websocket-server-to-play-chess)
+- [Chess moves](#chess-moves)
+- [Features](#features)
+- [Limitations](#limitations)
+- [References](#references)
+
+## Local setup
+
+There are two major ways to locally setup the project to have it up and running:
+
+- [Build from source](#build-from-source)
+- [Build with Docker](#build-from-docker)
+
+### Build from source
```

Comment:
Running the instructions below, I don't get the same result (the UUID) that I get with the Docker section.

```
╰─± go run .
2023/06/05 16:05:57 running chess server on port :8080...

```